testsuite/pytests/test_spatial/test_connect_sliced.py

```diff
@@ -131,7 +131,10 @@ def test_sliced_spatial_inheritance(self):
         extent_pos = nest.spatial.free(nest.random.uniform(-0.5, 0.5), extent=[5., 2.5])
         for pos in [self.free_pos, self.grid_pos, wrapped_pos, extent_pos]:
             nest.ResetKernel()
-            nodes = nest.Create('iaf_psc_alpha', positions=pos)
+            # Don't specify number of nodes if using grid_pos
+            nodes = (nest.Create('iaf_psc_alpha', positions=pos)
+                     if pos == self.grid_pos else
+                     nest.Create('iaf_psc_alpha', 10, positions=pos))
```

Comment:
To make the code consistent with the comment, I would turn the statement around:
```suggestion
            # Don't specify number of nodes if using grid_pos
            nodes = (nest.Create('iaf_psc_alpha', 10, positions=pos)
                     if pos != self.grid_pos else
                     nest.Create('iaf_psc_alpha', positions=pos))
```

Since assigning to `nodes` is more of a side effect and the main thing is creating, I wonder if it wouldn't be more readable to write an explicit if-else statement:

```suggestion
            if pos == self.grid_pos: 
                # Don't specify number of nodes if using grid_pos
                nodes = (nest.Create('iaf_psc_alpha', positions=pos)
            else:
                nodes = nest.Create('iaf_psc_alpha', 10, positions=pos))
```